$NetBSD: patch-me,v 1.7 2012/04/28 22:48:06 ryoon Exp $

Add DragonFly support.
Disable assembly routines in freebl on Linux (PR pkg/46056)

--- mozilla/security/nss/lib/freebl/Makefile.orig	2012-04-23 06:28:18.000000000 +0000
+++ mozilla/security/nss/lib/freebl/Makefile
@@ -279,7 +279,7 @@ endif
 # to bind the blapi function references in FREEBLVector vector
 # (ldvector.c) to the blapi functions defined in the freebl
 # shared libraries.
-ifeq (,$(filter-out BSD_OS FreeBSD Linux NetBSD OpenBSD, $(OS_TARGET)))
+ifeq (,$(filter-out BSD_OS DragonFly FreeBSD Linux NetBSD OpenBSD, $(OS_TARGET)))
     MKSHLIB += -Wl,-Bsymbolic
 endif
 
